packages/manager/modules/core/src/index.js

```diff
@@ -85,6 +86,10 @@ angular
 
       $translateProvider.use(defaultLanguage);
       $translateProvider.fallbackLanguage(LANGUAGES.fallback);
+
+      ouiCalendarConfigurationProvider.setLocale(
```

Comment:
Must be moved to `packages/manager/modules/core/src/translate/ovh-ui-angular/index.js`.